src/Security/Security/help/Get-AzSqlVulnerabilityAssessmentBaselineRule.md

```diff
@@ -0,0 +1,209 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version:
```

Comment:
Please add online version, its format should be https://docs.microsoft.com/powershell/module/{ModuleName}/{CmdletName}

Reply:
pending